contracts/interfaces/0.8.x/IFilteredMinterV3.sol

```diff
@@ -0,0 +1,223 @@
+    event ConfigValueSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Adds value `_value`
+     * to the set of bytes32 at key `_key` for project `_projectId`.
+     */
+    event ConfigValueAddedToSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+
+    /**
+     * @notice Generic project minter configuration event. Removes value
+     * `_value` to the set of bytes32 at key `_key` for project `_projectId`.
+     */
+    event ConfigValueRemovedFromSet(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bytes32 _key,
+        bytes32 _value
+    );
+    ///////////////////////////
+    /**
+     * @notice Price per token in wei updated for project `_projectId` to
+     * `_pricePerTokenInWei`.
+     */
+    event PricePerTokenInWeiUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        uint256 indexed _pricePerTokenInWei
+    );
+
+    /**
+     * @notice Currency updated for project `_projectId` to symbol
+     * `_currencySymbol` and address `_currencyAddress`.
+     */
+    event ProjectCurrencyInfoUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        address indexed _currencyAddress,
+        string _currencySymbol
+    );
+
+    /// togglePurchaseToDisabled updated
+    event PurchaseToDisabledUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        bool _purchaseToDisabled
+    );
+
+    /**
+     * @notice Local max invocations for project `_projectId`, tied to core contract `_coreContractAddress`,
+     * updated to `_maxInvocations`.
+     */
+    event ProjectMaxInvocationsLimitUpdated(
+        uint256 indexed _projectId,
+        address indexed _coreContract,
+        uint256 _maxInvocations
+    );
+
+    // getter function of public variable
+    function minterType() external view returns (string memory);
+
+    function minterFilterAddress() external returns (address);
+
+    // Triggers a purchase of a token from the desired project, to the
+    // TX-sending address.
+    function purchase(
+        uint256 _projectId,
+        address _coreContract
+    ) external payable returns (uint256 tokenId);
+
+    // Triggers a purchase of a token from the desired project, to the specified
+    // receiving address.
+    function purchaseTo(
+        address _to,
+        uint256 _projectId,
+        address _coreContract
+    ) external payable returns (uint256 tokenId);
+
+    // Toggles the ability for `purchaseTo` to be called directly with a
+    // specified receiving address that differs from the TX-sending address.
+    function togglePurchaseToDisabled(
+        uint256 _projectId,
+        address _coreContract
+    ) external;
```

Comment:
I wonder if we want to continue to expose this in this interface here, vs. moving it to the `SharedMerkle` and related token-gated interfaces?